test_run_handler.go

```diff
@@ -119,11 +125,21 @@ func handleGet(w http.ResponseWriter, r *http.Request) {
                 runs[r.Revision] = make([]TestRun, 0)
             }
             runs[r.Revision] = append(runs[r.Revision], r)
+            if r.CreatedAt.Before(createdAtLimit) {
+                createdAtLimit = r.CreatedAt
+            }
         }
     }
 
+    var orderedSHAs []string
+    orderedSHAs, err = getOrderedSHAs(ctx, createdAtLimit)
+    if err != nil {
+        http.Error(w, err.Error(), http.StatusInternalServerError)
+        return
+    }
+
     // Serialize the data + pipe through the test-runs.html template.
-    testRunsBytes, err := json.Marshal(runs)
+    testRunsBytes, err := marshalInOrder(runs, orderedSHAs)
```

Comment:
JavaScript does not guarantee key order for objects - this approach might be easier done in JS.

Reply:
I think the idea is that a human reader of the API response can see things in the order they expect. @lukebjerring is that right?

Reply:
Done in https://github.com/w3c/wptdashboard/pull/211
@mdittmer yes, in theory JSON human-readability makes this change separately valuable even if #211 fixes https://github.com/w3c/wptdashboard/issues/208